app/Data/models/comment_like.py

```diff
@@ -15,7 +16,12 @@ class CommentLike(Base):
         sa.ForeignKey('comment.Id'),
         primary_key=True,
         nullable=False)
-    Liked = sa.Column(sa.Boolean, nullable=False)
+    Liked = sa.Column(
+        sa.Boolean,
+        nullable=False)
+
+    comment = relationship('Comment', back_populates='comment_like')
+    user = relationship('Comment', back_populates='comment_like')
```

Comment:
user = relationship('Comment', back_populates='comment_like')
should be
user = relationship('User', back_populates='comment_like')